5-Cookies/server/public/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
I haven't tested it but it's here for heroku to serve the client side, but since I didn't uploaded my server app to heroku I don't use this folder, so I will delete it.